core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-reports/components/dot-experiments-reports-chart/dot-experiments-reports-chart.component.ts

```diff
@@ -34,16 +36,19 @@ export class DotExperimentsReportsChartComponent implements OnChanges {
     @Input()
     isLoading = true;
     @Input()
-    config: { xAxisLabel: string; yAxisLabel: string; title: string };
+    config: { xAxisLabel: string; yAxisLabel: string };
     @Input()
     data: ChartData<'line'>;
 
+    protected chartId = `chart-` + getRandomUUID();
+
     protected readonly plugins = [htmlLegendPlugin];
 
     ngOnChanges(changes: SimpleChanges): void {
         const { config, data } = changes;
         if (config?.currentValue && data.currentValue) {
             this.options = getDotExperimentLineChartJsOptions({
+                legendId: this.chartId,
```

Comment:
do we really need the `legendId ` now, since the legend selector is now based on `chart.canvas`

Reply:
Correct, no need for it.